doc/src/Build_cmake.rst

```diff
@@ -22,27 +22,31 @@ Makefile(s). Example:
    cd lammps                        # change to the LAMMPS distribution directory
    mkdir build; cd build            # create a new directory (folder) for build
    cmake [options ...] ../cmake     # configuration with (command-line) cmake
-   make                             # compilation
+   make                             # compilation (or use "cmbuild" from "tools/cmake")
 
-The cmake command will detect available features, enable selected
+The ``cmake`` command will detect available features, enable selected
 packages and options, and will generate the build environment.  By default
 this build environment will be created for "Unix Makefiles" on most
 platforms and particularly on Linux.  However, alternate build tools
 (e.g. Ninja) and project files for Integrated Development Environments
 (IDEs) like Eclipse, CodeBlocks, or Kate can be generated, too. This is
-selected via the "-G" command line flag. For the rest of the documentation
+selected via the ``-G`` command line flag. For the rest of the documentation
 we will assume that the build environment is generated for makefiles
-and thus the make command will be used to compile and link LAMMPS as
-indicated above, producing (by default) an executable called "lmp" and
-a library called "liblammps.a" in the "build" folder.  When generating
+and thus the ``make`` command will be used to compile and link LAMMPS as
```

Comment:
Do we explicitly need to mention `make` (or `ninja`)? `cmake --build . -j N` is all the user really needs to know.

Reply:
Yes, because `cmake --build . -j N` an ugly command line "monster". Hence my desire to write the wrapper script that acts a bit like make.

Reply:
The `-j N` is optional. And `cmake --build .` is not too long! **I** just think the script is a little over the top for replacing a single command. @rbberger, what do you think?